pygfx/materials/_line.py

```diff
@@ -101,26 +124,41 @@ def map(self, map):
 
 
 class LineThinMaterial(LineMaterial):
-    """A simple line, drawn with line_strip primitives that has a thickness
+    """Thin line material.
+
+    A simple line, drawn with line_strip primitives that has a thickness
     of one physical pixel (the thickness property is ignored).
 
     You probably want to avoid this material, because its width is
     inconsistent and looks *very* thin on HiDPI monitors.
     """
 
+    # @almarklein: The docs advise to avoid this. Depricate before 1.0?
+
 
 class LineThinSegmentMaterial(LineMaterial):
-    """Simple line segments, drawn with line primitives that has a thickness
+    """Thin line segment material.
+
+    Simple line segments, drawn with line primitives that has a thickness
     of one physical pixel (the thickness property is ignored).
 
     You probably want to avoid this material, because its width is
     inconsistent and looks *very* thin on HiDPI monitors.
     """
 
+    # @almarklein: The docs advise to avoid this. Depricate before 1.0?
+
 
 class LineSegmentMaterial(LineMaterial):
-    """A material that renders line segments between each two subsequent points."""
+    """Line segment material.
+
+    A material that renders line segments between each two subsequent points."""
 
 
 class LineArrowMaterial(LineSegmentMaterial):
-    """A material that renders line segments that look like little vectors."""
+    """Arrow (vector) line material.
+
+    A material that renders line segments that look like little vectors."""
+
+    # @almarklein: This is a thin (1:1) wrapper around LineMaterial. Out of
+    # curiosity, how does it do more than the base class?
```

Comment:
Because the shader associated with this material does something different.

Reply:
Ah! so all the classes here that have an empty body have a shader in `pygfx.renderers.wgpu` that does something special when it encounters this class? I think this is something we should document in the narrative docs.

Reply:
This is how it works for all materials, yes - the shader is selected based on the class of the material, and a shader may also make choices based on the subclass and/or their properties.